vaadinfrontend/frontend/themes/datamanager/components/dialog.css

```diff
@@ -0,0 +1,57 @@
+.dialog {
+}
+
+.dialog::part(overlay) {
+  width: 66vw;
+}
```

Comment:
Is this a requirement? Do we want all dialogs to have the same width?

Reply:
I'm unsure, I will change it when I work on the others. Or should I add it explicitly to the project creation dialog instead?